pubspec.yaml

```
@@ -1,21 +1,22 @@
 name: flame_splash_screen
 description: Style your flame/flutter game with a beautiful splash screen. Simple to use but very customizable.
-version: 0.0.1
+version: 0.1.0
 homepage: https://github.com/flame-engine/flame_splash_screen
 
 environment:
-  sdk: ">=2.2.2 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
 
 dev_dependencies:
-  mockito: 4.1.1
+  mockito: ^5.0.9
   flutter_test:
     sdk: flutter
 
 flutter:
   assets:
-   - assets/flame-logo-white.gif
-   - assets/flame-logo-black.gif
+   - assets/layer1.png
+   - assets/layer2.png
+   - assets/layer3.png
```

```suggestion
   - assets/
```

also maybe we should put all images in an `images` folder? to follow flame conventions